### PR TITLE
✨Add new `ad-refresh` trigger

### PR DESCRIPTION
**Pull request by calebcordry**
Follow up to #16689 

It seems like the `unload` portion of the previous PR may not be necessary. This pulls out just the `ad-refresh` trigger which is fired on the `refresh()` method in `a4a`.  

Is this enough to satisfy our requirements?

### CODE REVIEW

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -923,6 +925,10 @@ export class AmpA4A extends AMP.BaseElement {
         return;
       }
       return this.mutateElement(() => {
+        // Fire an ad-refresh event so that 3rd parties can track when an ad
+        // has changed.
+        this.maybeTriggerAnalyticsEvent_('adRefresh');
```
Comment (calebcordry) [inline extensions/amp-a4a/0.1/amp-a4a.js]: Copying same question from previous PR: Do we need any extra variables here, or are the vars introduced by `maybeTriggerAnalyticsEvent_` sufficient?

Comment (lannka) [reply extensions/amp-a4a/0.1/amp-a4a.js]: or should you just call `triggerAnalyticsEvent` here directly? Are those `analyticsVars` really relevant here?

Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js]: I wasn't sure the exact use case. If you think it is unnecessary I can definitely call `triggerAnalyticsEvent` directly.

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 2]
```diff
@@ -145,6 +145,7 @@ export const AnalyticsTrigger = {
   AD_RENDER_START: 'ad-render-start',
   AD_RENDER_END: 'ad-render-end',
   AD_IFRAME_LOADED: 'ad-iframe-loaded',
+  AD_REFRESH: 'ad-refresh',
```
Comment (lannka) [inline extensions/amp-a4a/0.1/amp-a4a.js]: Make a comment that this is not part of normal lifecycle triggers.

Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js]: Done.

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js]: Done.

**Pull request merged**